src/main/java/com/scriptbasic/interfaces/NestedStructureHouseKeeper.java

```diff
@@ -10,6 +10,24 @@
  * date June 8, 2012
  */
 public interface NestedStructureHouseKeeper {
+    
+    static public enum EndOfStatementResult {
+        /**
+         * End of statement was consumed. Other processor should not be called.
```

Comment:
The word "processor" implies that there are only two processors. Better we can use the plural "processors".